connector/connect/server/src/main/scala/org/apache/spark/sql/connect/service/SparkConnectService.scala

```diff
@@ -85,7 +86,7 @@ class SparkConnectService(debug: Boolean)
             .setDomain("org.apache.spark")
             .putMetadata("classes", compact(render(allClasses(st.getClass).map(_.getName))))
             .build()))
-      .setMessage(StringUtils.abbreviate(st.getMessage, 2048))
+      .setMessage(if (message != null) message else "")
```

Comment:
I am wondering when `message` could be `null` than an empty string? If this is a valid case we can use `st: Optiona[Throwable]` instead?

Reply:
> I am wondering when message could be null than an empty string?

`NullPointerException`

> If this is a valid case we can use st: Optiona[Throwable] instead?

Can't, like NPE, the `Optiona[Throwable]` would not empty, but message in NPT still `null`.

Reply:
We should still abbreviate this message when it's not null